src/providers/OrganizationProvider.tsx

```diff
@@ -32,6 +32,7 @@ export default function OrganizationProvider({ children }: OrganizationProviderP
   const [orgPreferenceForId, setOrgPreferenceForId] = useState<number>(defaultSelectedOrg.id);
   const [orgAPIRequestStatus, setOrgAPIRequestStatus] = useState<APIRequestStatus>(APIRequestStatus.AWAITING);
   const [organizations, setOrganizations] = useState<Organization[]>();
+  const [acceleratorOrganizations, setAcceleratorOrganizations] = useState<AcceleratorOrganization[]>();
```

Comment:
It seemed like this provider might be the best place for this, but I'm happy to move it if we'd prefer to.